src/js/player.js

```diff
@@ -2664,6 +2664,16 @@ class Player extends Component {
     this.trigger('exitFullWindow');
   }
 
+  /**
+   * Handle keydown events from any Component of this player which is not
+   *  andled by that Component; allows for player-wide keyboard shortcuts
+   *
+   * @param {EventTarget~Event} event
+   *        The `keydown` event that caused this function to be called.
+   */
+  handleKeyPress(event) {
```

Comment:
this is empty for now but eventually we can add other things like `f` going to fullscreen?

Reply:
Yes. See https://github.com/videojs/video.js/issues/4048#issuecomment-437499539; I have a separate PR which builds on top of this one to do exactly that, but wasn't sure if you'd want it in the core or in a plugin.

Reply:
I think the hotkeys plugin is pretty popular, it's probably something we want included. Would be nice to figure out how to make it pluggable so that it can be extended with custom shortcuts as well. Though, that can happen down the line.